Src/WitsmlExplorer.Frontend/components/Modals/LogPropertiesModal.tsx

```diff
@@ -66,41 +66,34 @@ const LogPropertiesModal = (props: LogPropertiesModalInterface): React.ReactElem
                 id="uid"
                 label="uid"
                 value={editableLogObject.uid}
-                fullWidth
                 disabled={editMode}
                 required
-                error={!validText(editableLogObject.uid)}
                 helperText={editableLogObject.uid.length === 0 ? "A wellbore uid must be 1-64 characters" : ""}
-                inputProps={{ minLength: 1, maxLength: 64 }}
-                onChange={(e) => setEditableLogObject({ ...editableLogObject, uid: e.target.value })}
+                variant={validText(editableLogObject.uid) ? undefined : "error"}
+                onChange={(e: any) => setEditableLogObject({ ...editableLogObject, uid: e.target.value })}
               />
               <TextField
                 id="name"
                 label="name"
                 defaultValue={editableLogObject.name}
-                error={editableLogObject.name.length === 0}
                 helperText={editableLogObject.name.length === 0 ? "A log name must be 1-64 characters" : ""}
-                fullWidth
-                inputProps={{ minLength: 1, maxLength: 64 }}
-                onChange={(e) => setEditableLogObject({ ...editableLogObject, name: e.target.value })}
+                variant={editableLogObject.name.length === 0 ? "error" : undefined}
+                onChange={(e: any) => setEditableLogObject({ ...editableLogObject, name: e.target.value })}
+              />
+              <TextField disabled id="serviceCompany" label="service company" defaultValue={editableLogObject.serviceCompany} />
+              <TextField disabled id="startIndex" label="start index" defaultValue={editableLogObject.startIndex} />
+              <TextField disabled id="endIndex" label="end index" defaultValue={editableLogObject.endIndex} />
+              <TextField disabled id="wellUid" label="well uid" defaultValue={editableLogObject.wellUid} />
+              <TextField disabled id="wellName" label="well name" defaultValue={editableLogObject.wellName} />
+              <TextField disabled id="wellboreUid" label="wellbore uid" defaultValue={editableLogObject.wellboreUid} />
+              <TextField disabled id="wellboreName" label="wellbore name" defaultValue={editableLogObject.wellboreName} />
+              <Autocomplete
+                label="index curve"
```

Comment:
Should be `disabled={editMode}`

Reply:
On the Textfields?

Reply:
See last commit

Reply:
Only Autocomplete, the same as before the changes.

Reply:
![image](https://user-images.githubusercontent.com/66632214/212306276-482fada1-56ef-4f48-9c6f-4dd993ec1186.png)

Reply:
Fixed now